.github/workflows/build.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       - run: npm run prettier-check
       - run: npm run eslint
       - run: npm run build
+      - run: npm run test
```

Comment:
Could we upload the coverage to Codecov so that the tests you wrote up show up in the statistic? You can use this repo as an example:
- https://sourcegraph.com/github.com/sourcegraph/extension-api-stubs/-/blob/.github/workflows/build.yml#L23:38
- https://sourcegraph.com/github.com/sourcegraph/extension-api-stubs/-/blob/package.json#L32-44
- https://sourcegraph.com/github.com/sourcegraph/extension-api-stubs/-/blob/package.json#L16